### ISSUE TITLE
Support both DHCP and DHCPv6 at the same time

### ISSUE THREAD

**Issue by scibi** [debops/ansible-dhcpd#7]
It would be nice to support both DHCPv4 and DHCPv6 at the same time. Probably it would require some custom init script since it's still not supported by Debian (https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=592539).

**Comment by drybjed** [comment]
@scibi Yeah, I'm aware of that issue and above Debian bug. I want to wait and see if the issue gets resolved after Jessie release, and if it is, either use a backported `isc-dhcp-server` package or incorporate the Stretchy solution into the Ansible role itself.
